src/annotation/index.ts

```diff
@@ -61,6 +61,10 @@ export class Annotations {
   setAnnotationCoordinates = (newCoordinates: XYPoint[]): void => {
     this.data[this.activeAnnotationID]["coordinates"] = newCoordinates;
   };
+
+  getData = (): AnnotationsDataArray => {
```

Comment:
for consistency should this not be `getAnnotations` to match `getActiveAnnotation`? Also, is the type not just `[]`?